md/bonita-docker-installation.md

```diff
@@ -1,10 +1,26 @@
-# Deploy a Bonita platform using docker
+# Deploy Bonita Runtime with Docker
 
-How to install and use the docker distribution of the Bonita platform.
+How to install and use the Bonita Runtime docker distribution.
 
-This guide assumes you are on a unix-based operating system.
-It also assumes you use the subscription version of the docker image. 
-For the community version, see the guide [on docker hub](https://hub.docker.com/_/bonita).
+This guide assumes you have a working docker environment.
+
+## Quick start
+To start the latest community release
+```
+docker run --name bonita -d -p 8080:8080 bonita
+```
+
+To start the latest subscription release
+>Access to Quay.io has to be requested through customer portal
+```
+docker login quay.io
+docker pull quay.io/bonitasoft/bonita-subscription:latest
+docker logout quay.io
+docker run --name=bonita -d -p 8080:8080 bonitasoft/bonita-subscription:latest
```

Comment:
Does not work without `quay.io/` on my computer:
```suggestion
docker run --name=bonita -d -p 8080:8080 quay.io//bonitasoft/bonita-subscription:latest
```

Reply:
I fixed (and in all other places...)